### PR TITLE
Release 20

### PR DESCRIPTION
**Pull request by mstajdohar**

### CODE REVIEW

**Comment by mstajdohar** [comment]
@dblenkus Please review the last commit.

**Comment by mstajdohar** [comment]
@jkokosar Please review the first two commits.

File: resolwe_bio/processes/reads_processing/markduplicates.py [hunk 1]
```diff
@@ -54,6 +53,7 @@ class Input:
                         'otherwise need to be decoded. Default is STRICT.',
             choices=[
                 ('STRICT', 'STRICT'),
+                ('LENIENT', 'LENIENT'),
```
Comment (jkokosar) [inline resolwe_bio/processes/reads_processing/markduplicates.py]: @romunov should this choice be also exposed in the workflow inputs?

Comment (romunov) [reply resolwe_bio/processes/reads_processing/markduplicates.py]: Yes, I missed that one.

Comment (mstajdohar) [reply resolwe_bio/processes/reads_processing/markduplicates.py]: We'll do that for the next release.

**Pull request merged**